client/my-sites/people/people-list-item/style.scss

```diff
@@ -85,18 +85,47 @@
 	z-index: z-index( 'root', '.people-list-item .card__link-indicator' );
 }
 
+.people-list-item.is-invite .people-profile__username {
+	@include breakpoint( '>480px' ) {
+		font-size: 18px;
+	}
+}
+
+.people-list-item.is-invite-details {
+	// Make extra space for invite status on small screens
+	@include breakpoint( '<480px' ) {
+		padding-bottom: 34px;
+	}
+}
+
 .people-list-item__invite-status {
 	align-self: center;
-	display: block;
-	flex-grow: 0;
 	flex-shrink: 0;
-	margin-right: 30px;
 	margin-left: 12px;
 
-	color: $alert-green;
+	&:not( .is-invite-details ) {
+		// Get out of the way of the link to the invite details page
+		margin-right: 30px;
+
+		// Hide on small screens
+		@include breakpoint( '<480px' ) {
+			display: none;
+		}
+	}
+
+	&.is-invite-details {
+		// Show below the name and role on small screens
+		@include breakpoint( '<480px' ) {
+			position: absolute;
+			bottom: 4px;
+			margin-left: 80px;
+		}
+	}
+
+	color: darken( $alert-green, 10% );
 
 	&.is-pending {
-		color: $gray;
+		color: darken( $gray, 20% );
```

Comment:
I personally prefer darker, but `$gray-text-min` is darkened 18% for WCAG 2.0 AA compliance.

Reply:
Let's use `$gray-text-min`. We should avoid having color calculations like this throughout our CSS... let's use variables where there is one that is close enough.

Reply:
Combinations of `lighten|darken( $gray, 10|20|30% )` are specifically called out as acceptable:

https://github.com/Automattic/wp-calypso/blob/88d841c6/assets/stylesheets/shared/_colors.scss#L20

Whether this is good practice is debatable... I'm going to go with `$gray-text-min` here as we discussed.